exporter_test.go

```diff
@@ -134,8 +134,8 @@ func testExporter(t *testing.T, when spec.G, it spec.S) {
 
 		exporter = &lifecycle.Exporter{
 			Buildpacks: []lifecycle.GroupBuildpack{
-				{ID: "buildpack.id", Version: "1.2.3"},
-				{ID: "other.buildpack.id", Version: "4.5.6", Optional: false},
+				{ID: "buildpack.id", Version: "1.2.3", API: "0.1"},
+				{ID: "other.buildpack.id", Version: "4.5.6", API: "0.1", Optional: false},
```

Comment:
Could we try using the latest buildpack api here too? `0.1` feels a little strange, because it's not a supported version. I think in practice the version doesn't actually matter, except for this test: https://github.com/buildpacks/lifecycle/blob/d9c7de5395c1ba66682b3e548f8955a11b265c17/exporter_test.go#L1124-L1125.

Reply:
Exactly. It doesn't matter. We just have to set it in order to not fail [here](https://github.com/buildpacks/lifecycle/blob/d9c7de5395c1ba66682b3e548f8955a11b265c17/exporter.go#L442).
I thought about setting it to `0.1` like the version is set to `1.2.3` or `4.5.6` although these are just random versions.
I can change it to something else.
@natalieparellano, what do you think?

Reply:
To me `0.1` reads like an old version of the api rather than a random value, which is why I'd lean towards just having it be the latest. Then we could remove the lines that set it to `0.5` in the test that I linked to.

Reply:
I don't have strong opinion on either approach so I'll make this change.

Reply:
Please see https://github.com/buildpacks/lifecycle/pull/498/commits/99f51baecde6a658cb9f5ee0651d6870b5cdeb5b